File: kaltura/api_v3/lib/types/enums/KalturaBatchJobStatus.php
<?php
/**
 * @package api
 * @subpackage enum
 */
class KalturaBatchJobStatus extends KalturaEnum
{
	const PENDING = 0;
	const QUEUED = 1;
	const PROCESSING = 2;
	const PROCESSED = 3;
	const MOVEFILE = 4;
	const FINISHED = 5;
	const FAILED = 6;
	const ABORTED = 7;
}
?>

File: kaltura/api_v3/generator/output/dotNet/KalturaClient/Enums/KalturaBatchJobStatus.cs
namespace Kaltura
{
	public enum KalturaBatchJobStatus
	{
		PENDING = 0,
		QUEUED = 1,
		PROCESSING = 2,
		PROCESSED = 3,
		MOVEFILE = 4,
		FINISHED = 5,
		FAILED = 6,
		ABORTED = 7,
	}
}
